content/02-learn/08-checkpointing.mdx

```diff
@@ -3,20 +3,23 @@ title: Checkpointing
 metaTitle: Checkpointing
 ---
 
-Via it’s Checkpointing System, This provides the Ethereum Classic blockchain with 51% attack resistance, giving protection to the chain and confidence to the user that the system is secure.
+The purpose of the checkpointing system is to provide the Ethereum Classic blockchain with 51% attack resistance,
+giving protection to the chain and confidence to its users that the system is secure.
+> Note: 51% attacks occur when some miners (more than 50% of the network's miners) collude to cause network disruption.
 
 ## Motivation
 
-Proof of work blockchains need high amounts of hash power (mining participation) in order to have a secure network. Ethereum Classic has undergone several 51% in the past months due to low mining participation. Among other proposed solutions, checkpointing helps mitigate these attacks while employing a set of external participants that assist in guaranteeing the ledger's properties.
-This solution is proposed as a temporary measure and once hash power has been regained, it will no longer be needed.
+Proof of work blockchains need high amounts of hash power (mining participation) to secure a network. Ethereum Classic has undergone several 51% attacks in the past due to low mining participation.  
+Checkpointing defends against these attacks by using a checkpointing federation of nodes to guarantee the blockchain's integrity. The checkpointing federation is a set of external participants that provides independent verification of a group of blocks.
```

Comment:
This a bit too light on details for my liking. I want to dig into this a bit more and come back to this.

Reply:
Some important details (and terms) from the checkpointing proposal are:
- checkpoints are stored on a separate network: the proposed OBFT-based "side chain"
- the checkpointing nodes may be a new group structure (as a "federation of nodes", either from a fixed committee of some kind or the result of a recurring election process), or alternatively based on a randomized selection from miners. I would avoid the term "external" as they'll be people heavily involved in ETC in any case.
- an alternative is also described in some detail, which requires miners to write the checkpoint blocks to the bitcoin blockchain, together with timestamps. In case of multiple checkpoints with the same block number, the oldest is chosen.

Bottom line: I think it's worth mentioning that the proposal goes a lot further than just proposing a committee to take control over the blockchain, and there's also a couple "cryptospeak" terms that can be used to quickly give readers an idea of the approach.

Reply:
Done.